mlflow/store/model_registry/sqlalchemy_store.py

```diff
@@ -32,6 +32,7 @@
     SqlModelVersion,
```

Comment:
Unsure of how to populate `aliases` fields here in `ModelVersion` and `RegisteredModel` payloads. Is this automatically done via foreign keys or can this be manually set?

Reply:
For example, we're seeing this error [here](https://github.com/mlflow/mlflow/actions/runs/4452242680/jobs/7819716224?pr=8055): 
```
E           mlflow.exceptions.MlflowException: 'SqlRegisteredModel' object has no attribute 'registered_model_aliases'

```

Reply:
See comment in `mlflow/store/model_registry/dbmodels/models.py`